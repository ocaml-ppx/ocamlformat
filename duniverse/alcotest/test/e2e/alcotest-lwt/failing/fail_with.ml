let test_lwt _switch () = Lwt.fail_with "should fail"

let () =
  let open Alcotest_lwt in
  Lwt_main.run
  @@ run __FILE__
       [
         ( "all",
           [
             test_case "one" `Quick test_lwt;
             test_case "two" `Quick (fun _ () -> Lwt.return_unit);
           ] );
       ]
