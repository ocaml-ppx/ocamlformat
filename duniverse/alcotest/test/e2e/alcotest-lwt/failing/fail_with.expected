Testing `test/e2e/alcotest-lwt/failing/fail_with.ml'.
This run has ID `<uuid>'.

> [FAIL]        all          0   one.
  [OK]          all          1   two.

┌──────────────────────────────────────────────────────────────────────────────┐
│ [FAIL]        all          0   one.                                          │
└──────────────────────────────────────────────────────────────────────────────┘
[failure] should fail

Logs saved to `<build-context>/_build/_tests/<test-dir>/all.000.output'.
 ──────────────────────────────────────────────────────────────────────────────

Full test results in `<build-context>/_build/_tests/<test-dir>'.
1 failure! in <test-duration>s. 2 tests run.
